nav-authsql/_run.detached.bat

```diff
@@ -1,7 +1,7 @@
 set /p host=<__privatereponame.txt
 set /p ImageName=<__imagename.txt
-set ContainerName=%ImageName:/=_%
+set ContainerName=%ImageName::=_%
```

Comment:
If I run one of the .bats that directly start the container, I get an error that : is not allowed in container names. This happens I think because you use ContainerName as value for the --name param of docker run. Does this work for you without my change?